_ont/ont-huawei-ma5671a-carlito.md

```diff
@@ -48,7 +48,20 @@ fw_setenv target oem-generic
 fw_printenv target=oem-generic
 ```
 
-## Setting HSGMII Mode
+## Getting/Setting Speed LAN Mode
+
+| Velue | Speed    |
+| ----- | -------- |
+| 4     | 1 Gbps   |
+| 5     | 2.5 Gbps |
```

Comment:
Actually sgmii_mode=5 doesn't force 2.5G mode but enables autonegotiation with 2.5G. The currently negotiated speed cannot be guaranteed to be 2.5G with sgmii_mode=5